### PR TITLE
fix: vulnerability fix

### PR DESCRIPTION
**Pull request by Berkmann18**
**What**:
Use more secure versions of dependencies.

**Why**:
To not have noise from dependency audits and potential issues from dependency vulnerabilities.

**How**:
Using `npm audit --fix`.

**Checklist**:

- [ ] Documentation
- [x] Ready to be merged 
- [ ] Added myself to contributors table. 
[Bot Usage](https://allcontributors.org/docs/en/bot/installation#4-update-your-contributing-documentation)

### CODE REVIEW

**Review by gr2m — APPROVED** [review]
The version change should make no difference for this package

File: package.json [hunk 1]
```diff
@@ -18,14 +18,13 @@
...
     "broken-link-checker": "^0.7.8",
     "cz-conventional-changelog": "^2.1.0",
     "docusaurus": "^1.7.3",
     "git-cz": "^4.1.0",
     "react": "^16.6.3",
-    "semantic-release": "^15.10.5"
+    "semantic-release": "^17.4.2"
   },
   "engines": {
     "node": ">=8",
```
Comment (gr2m) [inline package.json:30]: we need to bump the node version. I think we can safely bump it to 12

Comment (Berkmann18) [reply package.json:30]: Yeah, good point.

**Comment by Berkmann18** [comment]
Yeah, there were a bunch of vulnerabilities highlighted by GH (some of which are on https://github.com/all-contributors/all-contributors/security/dependabot) so I thought it would be wise to do this which should also publish translations and resolve 404's.

File: package.json (hunk 1 above)
Comment (Gam3356) [reply package.json]: The hacking have my email address jeromeparish30@gmail.com and they hacking my iCloud

**Pull request merged**